src/app/shared/components/chat-bot/chat-bot.component.html

```diff
@@ -0,0 +1,156 @@
+<div class="chat-bot">
+  <input
+    type="checkbox"
+    class="chat-bot__checkbox"
+    name="cb_chatbot"
+    id="cb_chatbot"
+  />
+  <div class="chat-bot__dialog">
+    <div class="chat-bot__dialog__message-container" #messageContainer>
+      <div
+        class="chat-bot__dialog__message"
+        [ngClass]="
+          message.sender === 'user'
+            ? 'chat-bot__dialog__message--incoming'
+            : 'chat-bot__dialog__message--outcoming'
+        "
+        *ngFor="let message of messages"
+      >
+        <div class="chat-bot__dialog__message__text">{{ message.content }}</div>
+        <div
+          class="chat-bot__dialog__message__products"
+          *ngIf="message.productsInfo && message.productsInfo.length > 0"
+        >
+          <div
+            class="chat-bot__dialog__message__product"
+            *ngFor="let product of message.productsInfo"
+            (click)="navigateProduct(product.id)"
+          >
+            <img
+              [src]="product.images[0]"
+              alt="Shoe image"
+              class="chat-bot__dialog__message__product__image"
+            />
+            <div class="chat-bot__dialog__message__product__info">
+              <div class="chat-bot__dialog__message__product__info__title">
+                {{ product.title }}
+              </div>
+              <div class="chat-bot__dialog__message__product__info__type">
+                {{ product.type | productType }}
+              </div>
+              <div class="chat-bot__dialog__message__product__info__sizes">
+                <div
+                  class="chat-bot__dialog__message__product__info__size"
+                  *ngFor="let size of product.sizes"
+                >
+                  {{ size }}
+                </div>
+              </div>
+              <div class="chat-bot__dialog__message__product__info__price">
+                {{ product.price | currency }}
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="chat-bot__dialog__message-input">
+      <form
+        class="chat-bot__dialog__message-input__form"
+        [formGroup]="messageForm"
+        *ngIf="messageForm"
+      >
+        <input
+          class="chat-bot__dialog__message-input__input"
+          type="text"
+          placeholder="Type message..."
+          [formControl]="getFormControl('message')"
+        />
+      </form>
+      <custom-button
+        [buttonClass]="'btn--circle btn--blue'"
+        (onClick)="sendMessage()"
+      >
+        <svg
```

Comment:
For SVGs in order to have component more readable, consider creating something like this, if that would work for you

https://medium.com/@rado.sabo/best-way-to-use-svg-icons-in-angular-with-caching-and-possibility-of-changing-colour-71923bb3f189